### PR TITLE
add outputstyle in property

### PR DESCRIPTION
**Pull request by sillydong**
plain style:
image
table style:
image
json style:
image

### CODE REVIEW

**Comment by sillydong** [comment]
@siddontang  please help review, thanks

File: pkg/util/output.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+// output style
+const (
+	OutputStylePlain = "plain"
+	OutputStyleTable = "table"
+	OutputStyleJson  = "json"
+)
+
+func RenderString(format string, headers []string, values [][]string) {
```
Comment (siddontang) [inline pkg/util/output.go:20]: @sillydong 

following the go style, can you add the comments for these public functions?

**Comment by sillydong** [comment]
code is updated @siddontang  please help review again. Thanks

**Review by dbsid — APPROVED** [review]
lgtm

**Pull request merged**